src/main/java/uk/gov/companieshouse/api/accounts/service/FilingService.java

```diff
@@ -1,6 +1,9 @@
 package uk.gov.companieshouse.api.accounts.service;
 
+import java.io.IOException;
+import uk.gov.companieshouse.api.accounts.model.entity.CompanyAccountEntity;
 import uk.gov.companieshouse.api.accounts.model.filing.Filing;
+import uk.gov.companieshouse.api.accounts.transaction.Transaction;
```

Comment:
Can you remove the unused imports please. Has Sonar been run against this PR?

Reply:
43107d8